test/azul_test_case.py

```diff
@@ -339,6 +342,19 @@ def _mock_dss_query_prefix(cls):
     def _restore_dss_query_prefix(cls):
         cls._dss_prefix_mock.stop()
 
+    _aws_account_name = 'test-hca-dev'
+
+    @classmethod
+    def _mock_lambda_env(cls):
+        cls._lambda_env_mock = patch.dict(os.environ,
+                                          AWS_LAMBDA_FUNCTION_NAME='unit-tests',
+                                          azul_aws_account_name=cls._aws_account_name)
```

Comment:
Please move this aspect (the mocking of `azul_aws_account_name`) into `_mock_aws_account_id` and rename that method to `_mock_aws_account`.